jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -193,6 +193,9 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
       else
         OPENJ9_BUILDSPEC="win_x86-64_cmprssptrs"
       fi
+    elif test "x$OPENJDK_BUILD_OS" = xmacosx; then
+      OPENJ9_PLATFORM_CODE=os64
```

Comment:
This will require adding `PLATFORM-os64` to the list of `platforms` in `jpp_configuration.xml` in openj9 can you open a PR for that first?

Reply:
in **openj9-openjdk-jdk9** extensions repo, `oa64` is used instead of `os64`: [code_link](https://github.com/ibmruntimes/openj9-openjdk-jdk9/blob/openj9/closed/autoconf/custom-hook.m4#L221).

also, we should add support for `cmprssptrs` here:
```
    ......
    elif test "x$OPENJDK_BUILD_OS" = xmacosx; then
      OPENJ9_PLATFORM_CODE=oa64
      if test "x$OPENJ9_LIBS_SUBDIR" = xdefault; then
        OPENJ9_BUILDSPEC="osx_x86-64"
      else
        OPENJ9_BUILDSPEC="osx_x86-64_cmprssptrs"
      fi
    ......
```

Reply:
Good catch on `oa` vs `os`.  The os is a typo